test/unit/config_report_status_calculator_test.rb

```diff
@@ -1,5 +1,5 @@
 require 'test_helper'
-class ReportStatusCalculatorTest < ActiveSupport::TestCase
+class ConfigReportStatusCalculatorTest < ActiveSupport::TestCase
```

Comment:
Not sure what happened here - I think you originally renamed the calculator to ConfigReportStatusCalculator as that's what it does, it calculates statuses of config reports.  I think in response to my comment at https://github.com/theforeman/foreman/pull/2750#discussion-diff-42617655 about the placement of the report_status method, you've renamed it back, but not the test?

I'm not sure I understand why you renamed it back.